lib/single-new-eks-opensource-observability-pattern/index.ts

```diff
@@ -173,12 +171,17 @@ export default class SingleNewEksOpenSourceobservabilityPattern {
             addOns.push(new blueprints.addons.IstioControlPlaneAddOn({
                 version: "1.18.2"
             }));
-            addOns.push(new IstioIngressGatewayHelmAddon);
-            addOns.push(new IstioCniHelmAddon);
+            addOns.push(new blueprints.addons.IstioIngressGatewayAddon({
+                version: "1.18.2"
```

Comment:
is specifying the exact version important here? I see other addons don't specify it, is it in this case different from the default?

Reply:
Yes this particular demonstration we are doing works only with this version or the example setup changes. So we are sticking to this version for now.